core/trino-main/src/main/java/io/trino/server/ui/OAuthIdTokenCookie.java

```diff
@@ -34,17 +32,14 @@ private OAuthIdTokenCookie() {}
 
     public static NewCookie create(String token, Instant tokenExpiration)
     {
-        return new NewCookie(
-                ID_TOKEN_COOKIE,
-                token,
-                UI_LOCATION,
-                null,
-                DEFAULT_VERSION,
-                null,
-                DEFAULT_MAX_AGE,
-                Date.from(tokenExpiration),
-                true,
-                true);
+        return new NewCookie.Builder(ID_TOKEN_COOKIE)
+                .value(token)
+                .path(UI_LOCATION)
+                .domain(null)
```

Comment:
Isn't domain null by default?

Reply:
it is, I wanted to reflect values explicitly so there is no confusion which values are default and which are not